### DISCUSSION TITLE
Which RS485 cable to use with Seplos BMS?

### DISCUSSION THREAD

**Discussion by andreas-bulling** [Louisvdw/dbus-serialbattery#812] [Q&A]
Hi,

I have a quick question as I found different recommendations online. For the connection between BMS (Seplos in my case) and the Cerbo - should I use the cable with the 2 wires or the fully connected ones? What's the difference between them?

**Comment by mdkeil** [comment]
RS485 needs 3 wires (A+, B-, GND).. I use [this](https://amzn.eu/d/hUlUaNd) adapter and a cutted network cable for the connection between Seplos BMS 10E and my raspberry Pi 2 with venusOS 3.10 + last nightly of the dbus-serial-battery-driver.
